src/components/AddItem/AddItem.js

```diff
@@ -1,7 +1,13 @@
 import React from 'react';
+import NavBar from '../NavBar/NavBar';
 
 const AddItem = () => {
-  return <div>Add an Item</div>;
+  return (
+    <>
```

Comment:
This is a nifty shorthand! I didn't realize empty brackets were the short syntax for `<React.Fragment>`! Learn something new every day 📚